src/broker/__tests__/fetch.spec.js

```diff
@@ -21,6 +21,16 @@ const maxBytesPerPartition = 1048576 // 1MB
 const maxWaitTime = 100
 const timestamp = 1509827900073
 
+expect.extend({
```

Comment:
Add this to `testHelpers/setup.js` instead, where we already introduce custom matchers.

Reply:
Good one, done with aacf5c2.